### ISSUE TITLE
Wikivoyage HomePage looks really broken

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#794]
Online:
![online](https://user-images.githubusercontent.com/1029718/59196985-6e160880-8b90-11e9-81cb-d491be6d49a8.png)

ZIM (via kiwix-serve):
![zim](https://user-images.githubusercontent.com/1029718/59197002-75d5ad00-8b90-11e9-8502-d9fa6e785990.png)

**Comment by kelson42** [comment]
Pretty sure this is a duplicate of #841. Will check

**Comment by kelson42** [comment]
After #841 fix, this is better, but we still have problem, at least:
* Big pointer image
* duplicate landscape images

![scr](https://user-images.githubusercontent.com/1029718/60387585-a6c94380-9aa5-11e9-9b99-346208d731aa.png)

**Comment by ISNIT0** [comment]
I don't think this is related to #841 
The WikiVoyage page uses a custom gadget for the carousel. At the moment, we use hard-coded scripts. (As we always have)

**Comment by kelson42** [comment]
@ISNIT0 It seems to me this bug might be not trivial to fix. Lets put it aside first and fix the other ones in the milestone.

**Comment by kelson42** [comment]
This seems to be fine now!
